### PR TITLE
fix: Default parameters and update documents

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [x] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [x] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Default parameters and update documents

## Issue this PR will close

close: #**issue_number**

### CODE REVIEW

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
this change using env-file breaks the command line override unless you change the envfile

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: docs_src/performance.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ make benchmark
 ```
 
 ```bash
-make PIPELINE_SCRIPT=yolov5s.sh PIPELINE_COUNT=2 benchmark 
+make PIPELINE_COUNT=2 benchmark
```
Comment (antoniomtz) [inline docs_src/performance.md:14]: do we need a section to explain how to specify a different pipeline other than the default?
or do we have that section already?

**Review by antoniomtz — APPROVED** [review]
LGTM

**Pull request merged**